apps/coscrad-frontend/src/components/dynamicViews/commands/CommandButton.tsx

```diff
@@ -1,8 +1,8 @@
-import { CommandInfo } from './CommandPanel';
+import { ICommandInfo } from '@coscrad/api-interfaces';
 
 const log = (data: unknown): void => console.log(JSON.stringify(data));
 
-export const CommandButton = ({ label, schema }: CommandInfo) => (
+export const CommandButton = ({ label, schema }: ICommandInfo) => (
     <button key={label} onClick={() => log(schema)}>
```

Comment:
This could be a MUI button

Reply:
Let's make styling \ visual design part of the requirements of a separate story.